# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1285788157 25200
#      Wed Sep 29 12:22:37 2010 -0700
# Node ID 3b8817c91e313ae541f441090cdceb961d27c00b
# Parent  c33b7054f1f96e49526c6f46b5129923d5a7a770
fntests.m: Break output text differently for better readability.

diff --git a/test/ChangeLog b/test/ChangeLog
--- a/test/ChangeLog
+++ b/test/ChangeLog
@@ -1,8 +1,12 @@
+2010-09-29  Rik <octave@nomad.inbox5.com>
+
+	* fntests.m: Break output text differently for better readability.
+
 2010-09-28  Rik <octave@nomad.inbox5.com>
 
 	* test/@Gork/set.m, test/fntests.m, test/test_func.m, test/test_io.m:
 	Untabify scripts.
 
 2010-09-27  Rik <octave@nomad.inbox5.com>
 
 	* fntests.m: Use single quotes for regex when possible.
diff --git a/test/fntests.m b/test/fntests.m
--- a/test/fntests.m
+++ b/test/fntests.m
@@ -245,24 +245,23 @@ try
       t1 = "were";
       t2 = "failures";
     else
       t1 = "was";
       t2 = "failure";
     endif
     printf ("\nThere %s %d expected %s (see fntests.log for details).\n",
             t1, dxf, t2);
-    puts ("\nExpected failures are known bugs.  Please help improve\n");
-    puts ("Octave by contributing fixes for them.\n");
+    puts ("\nExpected failures are known bugs.  Please help improve Octave\n");
+    puts ("by contributing fixes for them.\n");
   endif
   if (dsk > 0)
     printf ("\nThere were %d skipped tests (see fntests.log for details).\n", dsk);
-    puts ("Skipped tests are features that are disabled in this version\n");
-    puts ("of Octave because the needed libraries were not present when Octave\n");
-    puts ("was built.\n");
+    puts ("Skipped tests are features that are disabled in this version of Octave\n");
+    puts ("because the needed libraries were not present when Octave was built.\n");
   endif
 
   report_files_with_no_tests (files_with_tests, files_with_no_tests, ".m");
   report_files_with_no_tests (files_with_tests, files_with_no_tests, ".cc");
 
   puts ("\nPlease help improve Octave by contributing tests for\n");
   puts ("these files (see the list in the file fntests.log).\n\n");
 
